app/models/ability.rb

```diff
@@ -26,7 +26,7 @@ def initialize(user)
     end
 
     can :update_conference_preferences, Team do |team|
-      user.admin? || (user.student? && team.students.include?(user))
+      team.accepted? && (user.student? && team.students.include?(user))
```

Comment:
👍  could even be more simpler or am I wrong?
```ruby
team.accepted? && team.students.include?(user)
```
If a user does not have the role "student" he or she cannot be returned as the students of the team I guess.